.travis.yml

```diff
@@ -13,9 +13,11 @@ before_script:
 - ./node_modules/.bin/eslint assets/js/components/*.js
 - ./node_modules/.bin/eslint assets/js/constants/*.js
 - # ./node_modules/.bin/eslint assets/js/controllers/*.js
+- ./node_modules/.bin/eslint assets/js/controllers/e*.js
 - # ./node_modules/.bin/eslint assets/js/core/*.js
 - # ./node_modules/.bin/eslint assets/js/directives/*.js
 - ./node_modules/.bin/eslint assets/js/filters/*.js
+- ./node_modules/.bin/eslint assets/js/helpers/*.js
```

Comment:
The linter doesn't seem happy about the helper filename. Are `Helpers` actually a thing in Angular or should this just be a service? @jtormey thoughts?